pouchdb-core/index.d.ts

```diff
@@ -426,6 +431,10 @@ declare namespace PouchDB {
         on(event: 'created', listener: (dbName: string) => any): this;
         on(event: 'destroyed', listener: (dbName: string) => any): this;
 
+        debug: debug.IDebug;
+
+        adapter(name: string, adapter: any): void;
```

Comment:
Ok, I found in source, can we add 3rd option?
https://github.com/pouchdb/pouchdb/blob/master/packages/node_modules/pouchdb-core/src/setup.js#L35